samples/valueentity-counter/README.md

```diff
@@ -0,0 +1,111 @@
+and in particular the [JavaScript section](https://developer.lightbend.com/docs/akka-serverless/js-services/index.html)
+
+## Prerequisites
+
+No additional tooling is required for local development. To build and deploy to a cluster:
+
+- Docker; see https://docs.docker.com/engine/install/
+- Akka Serverless CLI (`akkasls`); see https://developer.lightbend.com/docs/akka-serverless/getting-started/set-up-development-env.html#_akka_serverless_cli
+
+## Building
+
+To build, at a minimum you need to generate and process sources, particularly when using an IDE.
+A convenience is compile your project:
+
+```
+npm install
+npm run build
+```
+
+## Testing
+
+Basic skeleton unit tests for each method in each service will be generated by the `build` phase, but these will need to be fleshed out with assertions as behavior is implemented.
+Mocha is the testing framework employed, and Chai is used for assertions. To execute the tests:
+
+```
+npm run test
+```
+
+These tests leverage the mock event-sourced entity and value-entity classes provided by `testkit.js`. These classes mimic the minimal required machinery to execute commands and handle events against a single entity for simple unit testing.
+In future, more complicated testing can leverage the Akka Serverless integration test-kit which runs the proxy inside docker (see [this example](https://github.com/lightbend/akkaserverless-framework/blob/master/javascript-sdk/integration-test/integration-testkit-test.js)).
+
+## Integration Test
+
+The integration tests are manually written and can be executed by running:
+
+```
+npm run integration-test
+```
+
+## Running Locally
+
+In order to run your application locally, you must run the Akka Serverless proxy. The included `docker-compose` file contains the configuration required to run the proxy for a locally running application.
+It also contains the configuration to start a local Google Pub/Sub emulator that the Akka Serverless proxy will connect to.
+To start the proxy, run the following command from this directory:
+
+### macOS and Windows
+
+```
+docker-compose up
+```
+
+### Linux
+
+> On Linux this requires Docker 20.10 or later (https://github.com/moby/moby/pull/40007),
+> or for a `USER_FUNCTION_HOST` environment variable to be set manually.
+
+```
+docker-compose -f docker-compose.yml -f docker-compose.linux.yml up
+```
+
+To start the application locally, use the following commands:
+
+> Be sure to have performed `npm install` for the first time!
+
+```
+npm run build && npm run start
+```
+
+With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://docs.lbcs.dev/js-services/proto.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`. For example, using `curl`:
```

Comment:
```suggestion
With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://developer.lightbend.com/docs/akka-serverless/js-services/proto.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`. For example, using `curl`:
```

Link should show production docs...

Reply:
Created https://github.com/lightbend/akkaserverless-npm-js/pull/60